.pre-commit-config.yaml

```diff
@@ -57,7 +57,7 @@ repos:
     rev: v3.3.1
     hooks:
     - id: pyupgrade
-      exclude: ^(cognite/client/_proto.*)|(cognite/dm_clients/.*)|(examples/.*)|(tests/test_dm_clients/.*)$
+      exclude: ^(cognite/client/_proto.*)|(cognite/pygen/dm_clients/.*)|(examples/.*)|(tests/test_dm_clients/.*)$
```

Comment:
This seems to be cloned from the SDK? Can remove the exclude that covers protobuf files